### PR TITLE
KZG Initialization Refactor

### PR DESCRIPTION
**Pull request by holgerd77**
Small refactor in preparation for a simplified KZG initialization on the EthereumJS side as discussed in the chat.

### CODE REVIEW

**Comment by acolytec3** [comment]
Have revised this slightly as follows:
- Removed setup path from `loadKZG`
- Added optional param to `loadTrustedSetup` with trusted setup dictionary type
- Loads mainnet trusted setup if alternative not provided

File: src/index.ts
Comment (g11tech) [inline src/index.ts]:
```suggestion
    const loadTrustedSetup = (trustedSetup: TrustedSetup = mainnetTrustedSetup) => {
```
small nit, and then we can remove the undefined check

File: src/index.ts
Comment (g11tech) [inline src/index.ts]:
```suggestion
 * @param setupPath Optional setup, otherwise official KZG setup from the KZG ceremony is used
```

**Review by acolytec3 — APPROVED** [review]
LGTM.  WIll merge once @g11tech approves

File: src/index.ts
Comment (g11tech) [inline src/index.ts]:
```suggestion
     * @param blob - a blob of data formatted as a flattened Uint8Array of 4096 big endian KZG field elements (32 bytes)
```
1 blob is around 160kb i think

File: src/index.ts
Comment (g11tech) [inline src/index.ts]:
```suggestion
     * @returns a KZG commitment corresponding to the input blob formatted as a 48 byte Uint8Array
```
commitment is 48 bytes

File: src/index.ts
Comment (g11tech) [inline src/index.ts]: again correction needed for what is blob and size

File: src/index.ts
Comment (g11tech) [inline src/index.ts]:
```suggestion
     * @param commitment - a KZG commitment corresponding to a blob formatted as a 48 byte Uint8Array
```

File: src/index.ts
Comment (g11tech) [inline src/index.ts]:
```suggestion
     * @returns a 48 bytes KZG proof corresponding to the blob and KZG commitment
```

File: src/index.ts
Comment (acolytec3) [inline src/index.ts]:
```suggestion
     * @param blob  - a blob of data formatted as a flattened Uint8Array of 4096 big endian KZG field elements
```

File: src/index.ts
Comment (g11tech) [inline src/index.ts]: same correction for blob/size

**Review by acolytec3 — COMMENTED** [review]
Have also updated the changelog and release so we can push an update on NPM.

**Pull request merged**
